src/EventLogExpert/MainPage.xaml.cs

```diff
@@ -62,6 +63,7 @@ public partial class MainPage : ContentPage
         _settingsState = settingsState;
         _traceLogger = traceLogger;
         _updateService = updateService;
+        _urlOpenService = urlOpenService;

```

Comment:
DI param can also be removed or build will fail, can also cleanup usings since we shouldn't need System.Diagnostics